pkgs/by-name/in/insync/package.nix

```diff
@@ -71,47 +81,42 @@ let
       mkdir -p $out
       cp -R usr/* $out/
 
-      # use system glibc
-      rm $out/lib/insync/{libgcc_s.so.1,libstdc++.so.6}
-
-      # remove badly packaged plugins
-      rm $out/lib/insync/PySide2/plugins/platforminputcontexts/libqtvirtualkeyboardplugin.so
-
-      # remove the unused vendor wrapper
-      rm $out/bin/insync
-
       runHook postInstall
     '';
 
     # NB! This did the trick, otherwise it segfaults! However I don't understand why!
     dontStrip = true;
   };
 
-in buildFHSEnvBubblewrap {
-  name = pname;
-  inherit meta;
+in buildFHSEnv {
+  inherit pname version meta;
 
   targetPkgs = pkgs: with pkgs; [
-    insync-pkg
+    cacert
     libudev0-shim
+    insync-pkg
   ];
 
-  runScript = writeShellScript "insync-wrapper.sh" ''
-    # QT_STYLE_OVERRIDE was used to suppress a QT warning, it should have no actual effect for this binary.
-    echo Unsetting QT_STYLE_OVERRIDE=$QT_STYLE_OVERRIDE
-    echo Unsetting QT_QPA_PLATFORMTHEME=$QT_QPA_PLATFORMTHEME
-    unset QT_STYLE_OVERRIDE
-    unset QPA_PLATFORMTHEME
+  extraInstallCommands = ''
+    cp -rsHf "${insync-pkg}"/share $out
+  '';
 
+  runScript = writeShellScript "insync-wrapper.sh" ''
     # xkb configuration needed: https://github.com/NixOS/nixpkgs/issues/236365
     export XKB_CONFIG_ROOT=${xkeyboard_config}/share/X11/xkb/
-    echo XKB_CONFIG_ROOT=$XKB_CONFIG_ROOT
 
-    # For debuging:
+    # For debugging:
     # export QT_DEBUG_PLUGINS=1
-    # find -L /usr/share -name "*insync*"
 
-    exec /usr/lib/insync/insync "$@"
+    if [ "$1" == start ]; then
+        shift
+        set -- start --no-daemon "$@"
+    elif [ "$1" == start_daemon ]; then
+        shift
+        set -- start "$@"
+    fi
+
+    LC_TIME=C exec /usr/lib/insync/insync "$@" &>>~/.config/Insync/out.txt
```

Comment:
this doesn't look correct to me.
```console
./results/insync/bin/insync help 
/nix/store/vd7cahahwmv0dwyzs2wc56hv0vjhqjim-insync-wrapper.sh: line 16: /home/anna/.config/Insync/out.txt: No such file or directory
```

Reply:
you could be right. let me test it when home folder of it is empty.

Reply:
Doing a `mkdir -p` now first.

Reply:
this has issues too because all the output is captured and there is no information conveyed to the user.  additionally, the out.txt file seems to get deleted when trying start the service and nothing is printed out either which makes troubleshooting impossible.

<details>
<summary>test run output</summary>
<pre>
[nix-shell:~/.cache/nixpkgs-review/pr-279426-1]$ ./results/insync/bin/insync

[nix-shell:~/.cache/nixpkgs-review/pr-279426-1]$ cat ~/.config/Insync/out.txt 
Usage: insync [OPTIONS] COMMAND [ARGS]...

  Command line interface for the Insync process. Insync is an application that
  syncs all your Google Drive & OneDrive accounts to your local filesystem. It
  integrates with your file manager so you can use your Google Drive or
  OneDrive files locally while getting the benefits of the cloud. Sync any
  local folder, sync multiple accounts and sync across Linux, Windows and Mac.

Options:
  --help  Show this message and exit.

Commands:
  account            Manage accounts.
  config             Manage app settings.
  conflict           Resolve conflicts, and set automatic resolution...
  error              Manage and resolve errors and items that require...
  export-option      Manage the export option (for Google-format files).
  get-proxy          Get current proxy settings.
  hide               Hide Insync
  ignore-rules       Manage rules for an account to ignore specific files from
                     syncing.
  open-cloud         Open the selected Google or OneDrive file in the Web...
  open-share-dialog  Open the share dialog for the selected Google or...
  pause              Pause syncing (for all accounts).
  quit               Quit Insync
  resume             Resume syncing (for all accounts).
  selective-sync     Selectively sync cloud files to local via an ncurses...
  set-proxy          Set proxy settings.
  show               Show Insync
  start              Start Insync
  status             Get the overall app status, or per-file sync status.
  version            Show version

[nix-shell:~/.cache/nixpkgs-review/pr-279426-1]$ ./results/insync/bin/insync start

[nix-shell:~/.cache/nixpkgs-review/pr-279426-1]$ cat ~/.config/Insync/out.txt 
cat: /home/anna/.config/Insync/out.txt: No such file or directory

</pre>
</details>

Reply:
You are right. I did further refinement of the workaround to address this.